docs/pages/configuration/dev/files/sync.mdx

```diff
@@ -240,18 +308,45 @@ dev:
 If multiple sync paths with `startContainer` exist, DevSpace will wait for all to finish before starting the container. You can combine `startContainer` sync configurations and non `startContainer` configurations to customize when the container is actually started.
 :::
 
+>>>>>>> upstream/v6-docs
 ### `onUpload.restartContainer`
 The `restartContainer` option expects a boolean which defines if DevSpace should restart the container every time either a single file or even a batch of files have been uploaded to the container using file sync.
 
 :::caution Restart Helper Required
+<<<<<<< HEAD
+Setting `restartContainer: true` requires to set `injectRestartHelper: true` for the image that is used to run the affected container. Otherwise, this option does not have any effect.
+=======
 Setting `restartContainer: true` requires to set `command: ["my", "container", "entrypoint"]`. Otherwise, DevSpace doesn't know which command to restart.
```

Comment:
```suggestion
Setting `restartContainer: true` requires you to set `command: ["my", "container", "entrypoint"]`. Otherwise, DevSpace will not know which command to restart.
```